mysite/lti/views.py

```diff
@@ -61,27 +86,28 @@ def lti_init(request, course_id=None, unit_id=None):
             AttributeError) as err:
         is_valid = False
         session['message'] = "{}".format(err)
+        LOGGER.error(err.__str__())
```

Comment:
Use `LOGGER.exception("Human-friendly message here")` instead. It will add exception info into a log.